curriculum/getChallenges.js

```diff
@@ -177,10 +177,17 @@ async function buildBlocks({ basename: blockName }, curriculum, superBlock) {
     __dirname,
     `./challenges/_meta/${blockName}/meta.json`
   );
+
   let blockMeta;
-  try {
-    blockMeta = require(metaPath);
+
+  if (fs.existsSync(metaPath)) {
+    // try to read the file, if the meta path does not exist it should be a certification.
+    // As they do not have meta files.
+
+    blockMeta = JSON.parse(fs.readFileSync(metaPath));
```

Comment:
```suggestion
    const blockMeta = JSON.parse(fs.readFileSync(metaPath));
```
Since you're here, we can fix this.